app/assets/src/components/ui/controls/LiveSearchPopBox.jsx

```diff
@@ -99,25 +94,38 @@ class LiveSearchPopBox extends React.Component {
   renderSearchBox = () => {
     const { placeholder, rectangular, inputClassName } = this.props;
     const { isLoading, value } = this.state;
+
     return (
-      <Input
-        fluid
-        className={cx(
-          cs.searchInput,
-          rectangular && cs.rectangular,
-          inputClassName
-        )}
-        icon="search"
-        loading={isLoading}
-        placeholder={placeholder}
-        onChange={this.handleSearchChange}
-        onKeyPress={this.handleKeyDown}
-        value={value}
-        disableAutocomplete={true}
-      />
+      <div onFocus={this.handleFocus} onBlur={this.handleBlur}>
+        <Input
+          fluid
+          className={cx(
+            cs.searchInput,
+            rectangular && cs.rectangular,
+            inputClassName
+          )}
+          icon="search"
+          loading={isLoading}
+          placeholder={placeholder}
+          onChange={this.handleSearchChange}
+          onKeyPress={this.handleKeyDown}
+          value={value}
+          disableAutocomplete={true}
+        />
+      </div>
     );
   };
 
+  handleFocus = _ => {
+    this.setState({ focus: true });
+  };
+
+  handleBlur = _ => {
+    // Give a chance for handleResultSelect to happen. Otherwise, the dropdown
+    // will disappear before the item onClick fires.
+    setTimeout(() => this.setState({ focus: false }), 100);
```

Comment:
another solution would be to use `mousedown` event instead of `onclick`, but `mousedown` is not supported by the item component